ranges/stackarrays.nim

```diff
@@ -0,0 +1,57 @@
+proc alloca(n: int): pointer {.importc, header: "<alloca.h>".}
+
+type
+  StackArray*[T] = ptr object
+    bufferLen: int
```

Comment:
I think `int` is overhead here, because its stack array even `int32` would be enough.